.github/workflows/release_linux.yml

```diff
@@ -34,14 +47,14 @@ jobs:
 
     - name: Set up Python ${{ env.Python_REQUIRED_VERSION }}
       id: setup-python
-      uses: jmarrec/setup-python@v5
+      uses: jmarrec/setup-python@v5.4.0
```

Comment:
Pick up the new python binaries, so that there are ubuntu arm64 ones to play with!

Reply:
Yes, very good.  Very good.